### PR TITLE
Add TWIRF5: Impl Days Rome

### PR DESCRIPTION
**Pull request by gandro**
image

I would like to publish this tomorrow (Nov 14) morning :)

### CODE REVIEW

**Comment by apiraino** [comment]
I'd merge this along [this one](https://github.com/RustFestEU/rome.rustfest.eu/pull/42), too.

File: _posts/2018-11-14-impl-days-rome.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+The `impl Days` Rome will take place on **Monday, November 26th and Tuesday, November 27th 2018** at [**Eurostars Roma Aeterna**](https://www.eurostarshotels.co.uk/eurostars-roma-aeterna.html). The address of this venue is *Via Casilina, 125 / Piazza del Pigneto 9a Rome 00176, Italy*!
+
+<p style="text-align:center;">
+    <a href="https://rome.rustfest.eu/about_impl_days/"><strong>🦀 Click here to learn more the <code>impl Days</code> Rome 🦀</strong></a>
+</p>
+
+Attendance to the `impl Days` is included in your RustFest Rome ticket. You can <a href="https://ti.to/asquera-event-ug/rustfest-rome-2018/">buy your RustFest Rome ticket here</a>.
+
+# Our Friends at Crypttape
```
Comment (apiraino) [inline _posts/2018-11-14-impl-days-rome.md]:
```suggestion
# Our Friends at Cryptape
```

File: _posts/2018-11-14-impl-days-rome.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+We are excited to have Cryptape sponsor us for the first time in Rome! 
+
+[Cryptape](https://www.cryptape.com) is dedicated to providing infrastructure and services for the crypto economy. Their mission is to build trust with technology.
+
+<div style="text-align: center">
+  <a href="https://www.cryptape.com">
+    <img alt="Crypttape Logo" src="https://rome.rustfest.eu/assets/sponsors/cryptape.png"/>
```
Comment (apiraino) [inline _posts/2018-11-14-impl-days-rome.md]:
```suggestion
    <img alt="Cryptape Logo" src="https://rome.rustfest.eu/assets/sponsors/cryptape.png"/>
```

**Review by apiraino — APPROVED** [review]

**Comment by apiraino** [comment]
greenlight for me

**Comment by gandro** [comment]
You're awesome, @apiraino :)

**Pull request merged**